Functions.Templates/Templates/OutlookMessageWebhookRefresher-CSharp/function.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+  "bindings": [
+    {
+      "name": "myTimer",
+      "type": "timerTrigger",
+      "direction": "in",
+      "schedule": "0 * * */2 * *"
```

Comment:
may want to provide user the option to change this at the time of template creation